xbmc/cores/AudioEngine/Sinks/AESinkDARWINIOS.cpp

```diff
@@ -575,6 +575,7 @@ static void EnumerateDevices(AEDeviceInfoList &list)
   //device.m_dataFormats.push_back(AE_FMT_S24LE3);
   //device.m_dataFormats.push_back(AE_FMT_S32LE);
   device.m_dataFormats.push_back(AE_FMT_FLOAT);
+  device.m_wantsIECPassthrough = true;
```

Comment:
hmm, why does a sink want IEC if it does not support passthrough at all? I think this is not final ....